__tests__/App-test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @format
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`react-test-renderer` import should occur before import of `../App`](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5444564)